app/src/main/java/com/github/pedrovgs/effectiveandroidui/ui/activity/MainActivity.java

```diff
@@ -16,121 +16,55 @@
 package com.github.pedrovgs.effectiveandroidui.ui.activity;
 
 import android.os.Bundle;
-import android.support.v4.app.Fragment;
+
 import com.github.pedrovgs.effectiveandroidui.R;
-import com.github.pedrovgs.effectiveandroidui.domain.tvshow.TvShow;
-import com.github.pedrovgs.effectiveandroidui.ui.fragment.TvShowCatalogFragment;
 import com.github.pedrovgs.effectiveandroidui.ui.fragment.TvShowDraggableFragment;
 import com.github.pedrovgs.effectiveandroidui.ui.fragment.TvShowFragment;
 import com.github.pedrovgs.effectiveandroidui.ui.presenter.TvShowUIModule;
+
 import java.util.LinkedList;
 import java.util.List;
-import javax.inject.Inject;
 
 /**
  * Core activity of this application. This activity receives the launch intent and works as core of
  * the sample application.
- *
- * Review how this activity uses fragments to decide how to implement navigation using the
- * Navigator
- * entity.
  */
-public class MainActivity extends BaseActivity implements TvShowCatalogFragment.Listener {
-
-  @Inject Navigator navigator;
+public class MainActivity extends BaseActivity {
 
-  private TvShowDraggableFragment tvShowDraggableFragment;
-  private TvShowFragment tvShowFragment;
+    private TvShowDraggableFragment tvShowDraggableFragment;
+    private TvShowFragment tvShowFragment;
 
-  @Override
-  protected void onCreate(Bundle savedInstanceState) {
-    super.onCreate(savedInstanceState);
-    setContentView(R.layout.activity_main);
-    initializeTvShowFragment();
-    initializeTvShowDraggableFragment();
-  }
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+        initializeTvShowFragment();
+        initializeTvShowDraggableFragment();
+    }
```

Comment:
Can you fix this indentation changes for all the pull request? If I'm not wrong, this project uses Square's checkstyle.